git_machete/client.py

```diff
@@ -1746,7 +1746,9 @@ def __derive_remote_and_github_org_and_repo(self) -> Tuple[str, Tuple[str, str]]
         if not org_and_repo_for_github_remote:
             raise MacheteException(
                 fmt('Remotes are defined for this repository, but none of them '
-                    'corresponds to GitHub (see `git remote -v` for details)'))
+                    'corresponds to GitHub (see `git remote -v` for details)\n'
```

Comment:
Any success in replicating the error? is it due to `git push` indeed?

Reply:
I think so 👯 , see comments above.